monitors/SyntheticPegMonitor.js

```diff
@@ -85,7 +85,7 @@ class SyntheticPegMonitor {
     const deviationError = this._calculateDeviationError(uniswapTokenPrice, cryptoWatchTokenPrice);
     // If the percentage error is greater than (gt) the threshold send a message.
     if (deviationError.gt(this.web3.utils.toBN(this.deviationAlertThreshold))) {
-      this.logger.error({
+      this.logger.warn({
```

Comment:
I changed this from `error` to `warn` as `error` should be reserved for application errors only. these log messages will still fire Pagerduty alerts but wont be tagged as an error.

Reply:
Agreed on these -- only one that we might want to keep as an error is the peg price volatility alert, since that one doesn't depend on the synthetic price and may signal a true time of stress for the market. Thoughts? Not super opinionated either way.

Reply:
Is that an error though? it's more of a warning than anything else in my mind. I'd favor consistency imho but am happy to change this if you want.

Reply:
SGTM, fine with keeping as is.